src/utils/Navigation.js

```diff
@@ -13,13 +13,20 @@ class Navigation {
           title: 'Reference',
           to: '/reference',
           icon: 'Receipt',
-          children: [],
+          children: [
+            { title: 'Centrapay Experiences', children: [] },
+            { title: 'Digital Assets', children: [] },
+            { title: 'App Integrations', children: [] },
+            { title: 'Merchant Integrations', children: [] },
+          ],
         },
         {
           title: 'Connections',
           to: '/connections',
           icon: 'Connections',
-          children: [],
+          children: [
+            { title: 'Farmlands', to: '/connections/farmlands', children: [] },
```

Comment:
A consequence of this change is that if we want clicking on a heading to route to a page that is not the first page in the heading, we have to explicitly define that page's route here.
I don't think this is an issue as we are heading towards defining the navigation structure more explicitly anyway.

Reply:
Isn't that already the behavior that exists?